README.md

```diff
@@ -26,12 +26,16 @@ Virtual Environment setup:
 Running møte:
  - Start `memcached` by typing `memcached` in the terminal and pressing Return (Enter) key.
  - Edit [config.py](./files/config.py) with necessary information and place the `config.py` in [`mote/`](./mote)
- - Modify the copied `config.py` to use memcached by setting `use_memcached = TRUE`
+ - Modify the copied `config.py` to use memcached by setting `use_memcached = TRUE` (Optional)
    - You can do it by running the following command in command line too: `sed -i '/use_memcached = / s/False/True/' mote/config.py`
  - Enter the Pipenv's newly created virtualenv's shell with `pipenv shell`
  - Run `python runmote.py`
 
-**Note :** Make sure `memcached` is running in the background.
+OR
+
+Using `docker-compose` to run møte:
+ - Edit [config.py](./files/config.py) with necessary information.
```

Comment:
How about asking to copy, [config.py](./files/config.py) in [mote/](./mote) first and then using it, as it'll be the case by default ?

As we're doing it [here](https://github.com/fedora-infra/mote/pull/156#discussion_r537334571)

Reply:
Please refrain for exploring project technicalities in the CONTRIBUTING.md.

If at all we are keeping such information, we would want to keep it in README.md.